deployments/pulse3d/services/pulse3d/src/main.py

```diff
@@ -250,6 +251,10 @@ async def _get_jobs(con, token, job_ids):
     return await get_jobs(con=con, account_type=account_type, account_id=account_id, job_ids=job_ids)
 
 
+def _string_version_to_numbers(version_string: string) -> List[int]:
```

Comment:
@NikitaPoly there is an existing package that handles comparing semvers https://pypi.org/project/semver/, we should use it instead of writing our own code to parse strings into ints